songs/3.sql

```diff
@@ -0,0 +1 @@
+SELECT name FROM songs ORDER BY duration_ms DESC LIMIT 5;
```

Comment:
Structure this queries like you did with the others.
```suggestion
SELECT name FROM songs
ORDER BY duration_ms DESC
LIMIT 5;
```

Reply:
Still open.